tofu/geom/_raytracing_tools.pyx

```diff
@@ -2078,7 +2078,7 @@ cdef inline void are_visible_vec_vec(double[:, ::1] pts1, int npts1,
         is_visible_pt_vec(pts1[0,ii], pts1[1,ii], pts1[2,ii],
                           pts2, npts2,
                           ves_poly, ves_norm,
-                          ind[ii,...], k, ves_lims,
+                          ind[ii,...], k[ii,...], ves_lims,
```

Comment:
ok, 
Just question before validating:

1. Did you double-check to make sure you're incrementing the good index ? 
To be consistent `k` has to be shaped like `ind`, right ?
Can you just add a docstr to the top-most function (i.e.:  `_GG.LOS_areVis_PtsFromPts_VesStruct()`, the first one visible by the user) to explain how `k` should be shaped ? ( `(pts1.shape[1], pts2.shape[1])` or `(pts2.shape[1], pts1.shape[1])` ?)

2. Did you check the same error was not existing elsewhere ? 
I guess that in `_GG.LOS_isVis_PtFromPts_VesStruct()` `k` is fine as a 1d array, can you confirm ?

Reply:
Sorry I jumped the gun with the PR, I'm going to add a short test in unit test and already updated the doc